### ISSUE TITLE
OSX BigSur Panic: Runtime Error

### ISSUE THREAD

**Issue by MattWilcox** [craftcms/nitro#256]
### Description

On a clean OSX 11.2.1, following the install instructions at https://craftcms.com/docs/nitro/2.x/installation.html gets me as far as nitro init; but something is horribly wrong at that point...

```
mattwilcox@Matts-iMac ~ % nitro init
Checking Nitro…
  … creating network ✓
  … pulling image ✓
  … creating volume ✓
  … creating proxy ✓
Checking network…
  ✓ network ready
Checking proxy…
  ✓ proxy ready
Checking databases…
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x16a22b5]

goroutine 1 [running]:
github.com/craftcms/nitro/command/apply.NewCommand.func2(0xc000390000, 0x1d96ad8, 0x0, 0x0, 0x195fbe0, 0x1d96ad8)
	/home/runner/work/nitro/nitro/command/apply/apply.go:235 +0x695
github.com/craftcms/nitro/command/initialize.NewCommand.func1(0xc000397340, 0x1d96ad8, 0x0, 0x0, 0x0, 0x0)
	/home/runner/work/nitro/nitro/command/initialize/initialize.go:107 +0x696
github.com/spf13/cobra.(*Command).execute(0xc000397340, 0x1d96ad8, 0x0, 0x0, 0xc000397340, 0x1d96ad8)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:850 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0x1d5aa20, 0x0, 0x174db80, 0xc000108058)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
main.main()
	/home/runner/work/nitro/nitro/cmd/nitro/main.go:11 +0x2a
mattwilcox@Matts-iMac ~ %
```

I have tried `nitro destroy` and running nitro init again, makes no difference.

### Additional info

- Nitro version: 2beta7
- DockerApp: 3.1.0
- OSX: 11.2.1
- Intel iMac 2020, i7 3.8Ghz

**Comment by jasonmccallister** [comment]
@MattWilcox can verify you have databases defined in `~/.nitro/nitro.yaml`?

**Comment by MattWilcox** [comment]
I've had to fall back to Nitro1 as I had to get some work done on this machine: but I'm pretty sure it was actually blank when I checked it at the time - as in the entire file was empty. I thought `nitro init` would just repopulate that though?

**Comment by jasonmccallister** [comment]
Hmm, we would not overwrite if there was an existing file - it would destroy your config. Version 1 and version 2 use two different config files, can you provide the output of the `nitro.yaml` here? If its empty you can delete and run `nitro init` and it should resolve your issue...

**Comment by MattWilcox** [comment]
Yep, nitro.yaml is completely blank. The file exists, but has no content.
Once the work day is done I'll try switching over to Nitro2 again, delete that file, and see how that goes. Thanks Jason.

**Comment by jasonmccallister** [comment]
Ok great! I will see if I can find out why that file is blank... that is the first time hearing about this issue.

**Comment by MattWilcox** [comment]
I'm trying to remember, but i think I messed up early on - there's no instruction to make sure that Docker is running after installing it, so I think I ran nitro init without having Docker running. It did error, but it may have been later down the line. Honestly can't remember what happened other than I had to go start Docker as it hadn't started itself post install, and maybe that got as far as creating a blank yaml?
